snapcraft/cli/store.py

```diff
@@ -149,7 +149,7 @@ def push(snap_file, release):
         snapcraft push my-snap_0.2_amd64.snap --release edge
         snapcraft push my-snap_0.3_amd64.snap --release candidate,beta
     """
-    click.echo("Pushing {}".format(os.path.basename(snap_file)))
+    click.echo("Pushing {}.".format(os.path.basename(snap_file)))
```

Comment:
add!r

Reply:
Can you answer this doubt I have, is this going to duplicate with the "upload widget" message?

Reply:
It will, if it doesn't use deltas (in case of deltas the widget message will show a delta file instead of the original snap). Perhaps we could change this first message to "Preparing to push" and leave "Pushing" to the upload widget?

Reply:
Output would become (changing to "Preparing to push" and adding quotes to all filenames):
```
Preparing to push 'xmdp_0+git.7a7cf60_amd64.snap'.
After pushing, an attempt will be made to release to 'edge'
Generating xdelta3 delta for 'xmdp_0+git.7a7cf60_amd64.snap'.
Pushing 'xmdp_0+git.7a7cf60_amd64.snap.xdelta3' [=========== // ===========] 100%
```